virttest/iscsi.py

```diff
@@ -970,10 +965,20 @@ def create_backstore(self):
             # create image disk
             if not os.path.isfile(self.emulated_image):
                 process.system("mkdir -p %s" % os.path.dirname(self.emulated_image))
+                # optimize the `create file` command
+                self.create_cmd = _get_create_command(
+                    self.emulated_image, self.emulated_size + self.unit, self.create_cmd
+                )
                 process.system(self.create_cmd)
```

Comment:
Could you add print info for the command to create a file? That is easy to debug.

Reply:
Yes, i want to know what command will be executed.